lib/players/human.rb

```diff
@@ -0,0 +1,10 @@
+module Players
+
+  attr_accessor :board
+  class Human < Player
+    # your code here
+    def move(board)
+      input = gets

```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets
end
```